module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -1900,7 +1900,7 @@ public function handleCancelRequest($cancelDetails, $type = 'Hold')
                 'ns1:CancelRequestItemResponse/' .
                 'ns1:UserId/ns1:UserIdentifierValue'
             );
-            $itemId = $itemId ?? $requestId;
+            $itemId = empty($itemId) ? $requestId : $itemId;
```

Comment:
Would `$itemId ?: $requestId` also have the desired effect here?

Reply:
Yes, I believe it's equivalent.